### PR TITLE
i18n(fr): Add `integration-guide/vercel` in French language

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description (required)
Add `integration-guide/vercel` in French language

#### Related issues & labels (optional)

- Linked to https://github.com/withastro/docs/issues/6052
- Suggested label: i18n

### CODE REVIEW

**Review by VoxelMC — APPROVED** [review]
Thank you for this too, Thomas!

`As a non-native speaker, I reviewed this PR using the Docs Translation Review Checklist and it LGTM!`

File: src/content/docs/fr/guides/integrations-guide/vercel.mdx [hunk 1]
```diff
@@ -0,0 +1,411 @@
...
+```
+
+## Configuration
+
+Pour configurer cet adaptateur, passez un objet à l'appel de la fonction `vercel()` dans `astro.config.mjs` :
+
+### Web Analytics
+
+**Type:** `VercelWebAnalyticsConfig`<br/>
+**Available for:** Serverless, Static<br/>
```
Comment (McFlyPartages) [inline src/content/docs/fr/guides/integrations-guide/vercel.mdx:113]:
```suggestion
**Disponible pour:** Serverless, Static<br/>
```

**Review by McFlyPartages — COMMENTED** [review]
Only one change

**Comment by thomasbnt** [comment]
Merci @McFlyPartages pour ce changement !

**Review by McFlyPartages — APPROVED** [review]
LGTM

File: src/content/docs/fr/guides/integrations-guide/vercel.mdx [hunk 2]
```diff
@@ -0,0 +1,411 @@
...
+githubIntegrationURL: 'https://github.com/withastro/astro/tree/main/packages/integrations/vercel/'
+category: adapter
+i18nReady: true
+---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro';
+import Since from '~/components/Since.astro';
+
+Cet adaptateur permet à Astro de déployer votre [site rendu `hybrid` ou `server`](/fr/core-concepts/rendering-modes/#rendu-à-la-demande) sur [Vercel](https://www.vercel.com/).
+
+Si vous utilisez Astro en tant que [constructeur de site statique](/fr/core-concepts/modes de rendu/#pré-rendu), vous n'avez pas besoin d'adaptateur.
```
Comment (Yan-Thomas) [inline src/content/docs/fr/guides/integrations-guide/vercel.mdx:14]:
```suggestion
Si vous utilisez Astro en tant que [constructeur de site statique](/fr/core-concepts/rendering-modes/#pré-rendu), vous n'avez pas besoin d'adaptateur.
```

File: src/content/docs/fr/guides/integrations-guide/vercel.mdx [hunk 3]
```diff
@@ -0,0 +1,411 @@
...
+  output: 'server',
+  adapter: vercel(),
+});
+```
+
+### Choix d'une méthode de déploiement
+
+Vous pouvez vous déployer vers différentes cibles :
+
+* `serverless` : SSR à l'intérieur d'une [fonction Node.js] (https://vercel.com/docs/concepts/functions/serverless-functions).
```
Comment (Yan-Thomas) [inline src/content/docs/fr/guides/integrations-guide/vercel.mdx:85]:
```suggestion
* `serverless` : SSR à l'intérieur d'une [fonction Node.js](https://vercel.com/docs/concepts/functions/serverless-functions).
```

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🚀

**Pull request merged**